mlflow/transformers/__init__.py

```diff
@@ -2670,11 +2679,99 @@ def is_uri(s):
         if not valid_uri:
             raise MlflowException(
                 "An invalid string input was provided. String inputs to "
-                "audio files must be either a file location or a uri.",
+                "audio files/image files must be either a file location or a uri.",
                 error_code=BAD_REQUEST,
             )
+    def _convert_image_input(self, data):
+            """
+            Conversion utility for decoding the base64 encoded bytes data of a raw imagefile when
+            parsed through model serving, if applicable. Direct usage of the pyfunc implementation
```

Comment:
This function looks 99% the same as _convert_audio_input. Can we extract the common functions out and refactor to make the function simpler?